rubygem-foreman_xen/rubygem-foreman_xen.spec

```diff
@@ -17,17 +17,18 @@
 
 Summary:    Provision and manage XEN Server from Foreman
 Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    0.2.4
+Version:    0.3.0
 Release:    1%{?foremandist}%{?dist}
 Group:      Applications/System
 License:    GPLv3
 URL:        http://github.com/theforeman/foreman-xen
 Source0:    http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 
-Requires:   foreman-compute >= 1.10.0
+Requires:   foreman-compute >= 1.11.0
```

Comment:
This should now be just `foreman`, as the main Fog dependency is no longer required - depending on Foreman core plus fog-xenserver is enough.